sql/item_basic.sql

```diff
@@ -646,16 +645,16 @@ INSERT INTO `item_basic` VALUES (729,0,'bloodwood_log','bloodwood_log',12,0,43,0
 INSERT INTO `item_basic` VALUES (730,0,'piece_of_bloodwood_lumber','bldwd_lumber',12,0,43,0,1387);
 INSERT INTO `item_basic` VALUES (731,0,'aquilaria_log','aquilaria_log',12,0,43,0,0);
 INSERT INTO `item_basic` VALUES (732,0,'kapor_log','kapor_log',12,0,43,0,0);
-INSERT INTO `item_basic` VALUES (733,0,'piece_of_kapor_lumber','kapor_lumber',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (734,0,'chunk_of_thokcha_ore','thokcha_ore',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (735,0,'thokcha_ingot','thokcha_ingot',12,0,0,1,0);
```

Comment:
take for instance INSERT INTO `item_basic` VALUES (12467,0,'wool_cap','wool_cap',1,2080,17,0,4234);